projects/demo/src/modules/pipes/currency/examples/import/template.md

```diff
@@ -1,3 +1,11 @@
 ```html
-<tui-input-number [postfix]="826 | tuiCurrency">Type a sum</tui-input-number>
+<tui-textfield>
+  <label tuiLabel>Type a sum</label>
+
+  <input
+    formControlName="testValue"
+    tuiInputNumber
+    [postfix]="826 | tuiCurrency"
```

Comment:
It also needs a space for postfix, let's use prefix here instead and USD for currency

Reply:
https://github.com/taiga-family/taiga-ui/pull/10160/commits/461c7b55be81155d28845417774cba975ff17907